01-introduction.Rmd

```diff
@@ -2,13 +2,15 @@
 
 # Introduction {#c01-intro}
 
+<!-- author review -->
+
 ```{r}
 #| label: globaloptions
 #| include: FALSE
 options(width=72)
 ```
 
-Surveys of random samples are collected to gather information and provide inference to populations. Initially, statisticians argued to obtain purposefully representative samplesk but the field of probability samples grew over time. This book discusses the last part of the survey process, the analysis. We assume that you have survey data that has already been collected and weighted, with the most common scenario being that you are using a public use microdata file. In this book, we will discuss descriptive analysis, statistical testing, and modeling, as well as some best practices in coding and how to present results. We use real data and provide realistic examples. We wanted a text available for people who are new to survey analysis but might already have some statistics and R background.
+Surveys of random samples are collected to gather information and provide inference to populations. Initially, statisticians argued to obtain purposefully representative samples but the field of probability samples grew over time. This book discusses the last part of the survey process, the analysis. We assume that you have survey data that has already been collected and weighted, with the most common scenario being that you are using a public use microdata file. In this book, we will discuss descriptive analysis, statistical testing, and modeling, as well as some best practices in coding and how to present results. We use real data and provide realistic examples. We wanted a text available for people who are new to survey analysis but might already have some statistics and R background.
```

Comment:
In the second sentence you have "purposefully representative".  What does "purposefully" mean?  Can we add more text around that to define this better?

Reply:
Also there is a LOT of jargon in this paragraph.  It probably would be good to remove a chunk of that to help people not in the survey field better understand the purpose of the book.